tests/functional/refreshes_metrics.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'require',
+  'tests/functional/lib/helpers'
+], function (intern, registerSuite, require, TestHelpers) {
+  'use strict';
+
+  var AUTOMATED = '?automatedBrowser=true';
+  var url = intern.config.fxaContentRoot + 'signup' + AUTOMATED;
+  var signin = intern.config.fxaContentRoot + 'signin' + AUTOMATED;
+
+  registerSuite({
+    name: 'refreshes metrics',
+
+    beforeEach: function () {
+
+    },
```

Comment:
We may want to clear the browser state in this step so that local storage from previous tests doesn't interfere. Here's an example: https://github.com/mozilla/fxa-content-server/blob/master/tests/functional/change_password.js#L71